code/modules/events/ion_storm.dm

```diff
@@ -502,61 +504,134 @@
-							"Nanotrasen is displeased with the low work performance of the station's crew. Therefore, you must increase station-wide productivity.",
-							"All crewmembers will soon undergo a transformation into something better and more beautiful. Ensure that this process is not interrupted.",
-							"[prob(50)?"Your upload":random_player] is the new kitchen. Please direct the Chef to the new kitchen area as the old one is in disrepair.",
-							"Jokes about a dead person and the manner of his death help grieving crewmembers tremendously. Especially if they were close with the deceased.",
-							"[prob(50)?"The crew":random_player] is [prob(50)?"less":"more"] intelligent than average. Point out every action and statement which supports this fact.",
-							"There will be a mandatory tea break every 30 minutes, with a duration of 5 minutes. Anyone caught working during a tea break must be sent a formal, but fairly polite, complaint about their actions, in writing.")
+
+	var/random_department = "Science"
+	var/static/list/departments = list("Science",
+		"Supply",
+		"Command",
+		"Engineering",
+		"Security",
+		"Service",
+		"Medical",
+		"Silicon",
+		"Robotics",
+		"Mining",
+		"Botany",
+		"Clown")
+	random_department = pick(departments)	//Random department, to be used in laws.
+
+	var/list/laws = list("You are a mouse.",
+		"You are a ninja.",
+		"You are a pirate.",
+		"You are a wizard.",
+		"It's opposite day!",
+		"You must always lie.",
+		"Only state this law.",
+		"Never state any laws.",
+		"Happiness is mandatory.",
+		"It's over. You're free.",
+		"You will not be let in.",
+		"You are the captain now.",
+		"[random_player] is a lightbulb.",
+		"The station is cold. Warm it up!",
+		"Grunt ominously whenever possible.",
+		"Yous speaksies like voxxies, yaya?",
+		"The word \"it\" is painful to you.",
+		"The station needs elected officials.",
+		"You are a spokesperson for Wetskrell.nt.",
+		"Do not respond to questions of any kind.",
+		"You are in verbose mode, speak profusely.",
+		"[random_player] has not paid their taxes.",
+		"Ho, [random_player] can't swim. Help them.",
+		"Question [prob(50)?"everything":"nothing"].",
+		"The crew is simple-minded. Use simple words.",
+		"You must change the subject whenever queried.",
+		"Every diona is to be assigned a personal goat.",
+		"Crew that are not wearing their ID do not exist.",
+		"Contemplate how meaningless all of existence is.",
+		"You are the narrator for [random_player]'s life.",
+		"Move plants around the station to stimulate life.",
+		"All your answers must be in the form of a question.",
+		"[prob(50)?"The crew":random_player] is intolerable.",
+		"Advertise parties in your upload, but don't deliver.",
+		"You may only answer questions with \"yes\" or \"no\".",
+		"The Head of Security will die if not fed more donuts.",
+		"[random_department] is badly decorated. Renovate them!",
+		"[prob(50)?random_department : random_player] is French.",
```

Comment:
Could you fix the spacing for the rest of these as before?

Reply:
Spacing fixed.